plugins/woocommerce-admin/client/marketing/data-multichannel/actions.ts

```diff
@@ -38,9 +43,41 @@ export const receiveRecommendedChannelsError = ( error: ApiFetchError ) => {
 	};
 };
 
+export const receiveCampaignsSuccess = ( success: {
+	payload: Array< Campaign >;
+	error: boolean;
+	meta: {
+		page: number;
+		perPage: number;
+		total?: number;
+	};
```

Comment:
💅 A bit of confusion here... is a success but it has error field?

Reply:
💅  Missed JSDOCS

Reply:
This is addressed in [`9feaf55` (#36735)](https://github.com/woocommerce/woocommerce/pull/36735/commits/9feaf55a6070e11ceb8633d38f22e1b51fa645c9), as mentioned in https://github.com/woocommerce/woocommerce/pull/36735#discussion_r1122208649.